### DISCUSSION TITLE
Unable to access the web interface of Security Onion

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#11949] [Unsupported Versions]
Hi, I installed security onion 2.3 in AWS instance Ubuntu  manually by the command in security onion documentation 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/a4436230-7bc6-4fba-b9ba-2fc63c7eb041)

By using the above commands i have installed security onion in AWS instance which is Ubuntu OS. As it is AWS instance because of that reason it was giving private IP address as web access URL, we can't access the web interface using private IP.  So i set a host-name and added public IP and host-name in /etc/hosts file in instance also and in local system, because of this steps i got host-name as web access URL  this i have tried and got the web interface of security-onion using host-name past week.

But now if i follow the same steps and run the command so-allow -a -i IP-address. I'm not able to access the web interface of security-onion. IDK why I'm able to access the web console of security onion past week with this same steps but not now 

Should i have to follow any other steps to get web interface! Give some idea to solve this problem.

**Comment by dougburks** [comment]
2.3 reaches End Of Life in less than 4 months:
https://blog.securityonion.net/2023/12/4-month-end-of-life-eol-reminder-for.html

Therefore, you might want to go ahead and move to 2.4 now:
https://docs.securityonion.net/en/2.4/installation.html

Since you're using AWS, you may want to consider using our Amazon Image:
https://docs.securityonion.net/en/2.4/cloud-amazon.html

**Comment by sandraimmaculate** [comment]
Hi, thanks for your Response. 
I have installed security onion 2.4 standalone in AWS Ubuntu instance i have created 2 NIC and selected a instance type with 4vcpu and 16 GB ram and created 200gb root volume. And installed security onion manually by the commands provided in the document.
 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/900ab224-ecfd-497f-a3cc-616b33bae5d2)

So i set a host-name and added public IP and host-name in /etc/hosts file in instance also and in local system
and in security onion configuration also i gave the host name as access URL, it installed successfully.

But I'm not able to access the console using the host-name and also with public IP I'm getting like this 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/640eb297-98e4-459a-bd77-7571bd086d0a)

And here is the so-status 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/52ee3716-4adc-4321-b917-62c804f1f348)

I don't know why I'm not able access the console 
so-firewall includehost analyst <IP ADDRESS> i also executed this command by replacing my ip-address
Please help me to solve this!

**Reply by dougburks** [reply]
> I have installed security onion 2.4 standalone in AWS Ubuntu instance i have created 2 NIC and selected a instance type with 4vcpu and 16 GB ram and created 200gb root volume. And installed security onion manually by the commands provided in the document.

Please be aware that you may be able to make Ubuntu work, but we no longer officially support it as of 2.4:
https://docs.securityonion.net/en/2.4/os.html#unsupported

The best option for running in AWS is to use our cloud image:
https://docs.securityonion.net/en/2.4/cloud-amazon.html

Even if you decide not to use our cloud image, you should still review that page as it may provide helpful clues like:
_Users evaluating Security Onion for the first time should consider choosing the other option and specifying the node’s public cloud IP._
